apps/analysis/tests/test_apis.py

```diff
@@ -740,19 +740,76 @@ def test_clone_analysis(self):
         user2 = self.create_user()
         project = self.create_project()
         project.add_member(user)
+        entry = self.create_entry(project=project)
+        entry1 = self.create_entry(project=project)
         analysis = self.create(Analysis, project=project, title="Test Clone")
+        pillar = self.create(AnalysisPillar, analysis=analysis, title='title1', assignee=user)
+        analytical_statement = self.create(
+            AnalyticalStatement,
+            analysis_pillar=pillar,
+            statement='Hello from here',
+            client_id='1'
+        )
+        self.create(
+            AnalyticalStatementEntry,
+            analytical_statement=analytical_statement,
+            entry=entry,
+            order=1,
+            client_id='1'
+        )
+        self.create(
+            DiscardedEntry,
+            entry=entry1,
+            analysis_pillar=pillar,
+            tag=DiscardedEntry.TagType.REDUNDANT
+        )
 
-        url = url = f'/api/v1/projects/{project.id}/analysis/{analysis.id}/clone-analysis/'
+        url = url = f'/api/v1/projects/{project.id}/analysis/{analysis.id}/clone/'
+        # try to post with no end_date
         data = {
             'title': 'cloned_title',
         }
         self.authenticate(user)
         response = self.client.post(url, data)
-        self.assert_201(response)
+        self.assert_400(response)
```

Comment:
Need to further check the error structure. Let's share the current error structure with @tnagorra.